### PR TITLE
Feature-SLIM-1115-set alarm notifications with all alarm notifications

### PR DESCRIPTION
**Pull request by bjornamkreutz**
SLIM-1115

### CODE REVIEW

**Comment by bjornamkreutz** [comment]
retest this please

File: cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/support/ws/smartmetering/configuration/SetAlarmNotificationsRequestFactory.java [hunk 1]
```diff
@@ -63,4 +74,19 @@ private static SetAlarmNotificationsRequestData fetchAlarmNotifications(
         return setAlarmNotificationsRequestData;
     }
 
+    private static AlarmNotification addAlarmNotification(final Map<String, String> requestParameters,
```
Comment (riannefijn) [inline cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/support/ws/smartmetering/configuration/SetAlarmNotificationsRequestFactory.java]: This method makes/gets an AlarmNotification, it doesn't add it. Rename it to make/getAlarmNotification

File: cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/support/ws/smartmetering/configuration/SetAlarmNotificationsRequestFactory.java [hunk 2]
```diff
@@ -46,14 +49,22 @@ public static SetAlarmNotificationsAsyncRequest fromScenarioContext() {
 
     private static SetAlarmNotificationsRequestData fetchAlarmNotifications(
             final Map<String, String> requestParameters) {
-        final AlarmNotification alarmNotification = new AlarmNotification();
-        alarmNotification.setAlarmType(AlarmType.valueOf(requestParameters.get(PlatformSmartmeteringKeys.ALARM_TYPE)));
-        alarmNotification
-                .setEnabled(Boolean.parseBoolean(requestParameters.get(PlatformSmartmeteringKeys.ALARM_TYPE_ENABLED)));
-
         final List<AlarmNotification> lstAlarmNotifications = new ArrayList<>();
```
Comment (riannefijn) [inline cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/support/ws/smartmetering/configuration/SetAlarmNotificationsRequestFactory.java]: While you're at it, could you add an i here? listAlarmNotifications (or alarmNotificationsList) is a better name in my opinion.

File: cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/support/ws/smartmetering/configuration/SetAlarmNotificationsRequestFactory.java [hunk 3]
```diff
@@ -49,18 +49,18 @@ public static SetAlarmNotificationsAsyncRequest fromScenarioContext() {
 
     private static SetAlarmNotificationsRequestData fetchAlarmNotifications(
             final Map<String, String> requestParameters) {
-        final List<AlarmNotification> lstAlarmNotifications = new ArrayList<>();
+        final List<AlarmNotification> AlarmNotificationsList = new ArrayList<>();
```
Comment (riannefijn) [inline cucumber-tests-platform-smartmetering/src/test/java/com/alliander/osgp/cucumber/platform/smartmetering/support/ws/smartmetering/configuration/SetAlarmNotificationsRequestFactory.java]: start with a lower case 'a'

**Pull request merged**